### ISSUE TITLE
Remove the `nm-applet` dependency?

### ISSUE THREAD

**Issue by monomadic** [ProtonVPN/linux-cli#62]
I'm running manjaro (but have run several distros with the same issues) and I always use a simple window manager (usually sway). The `protonvpn-cli` package never seems to document or require the `nm-applet` dependency, and this is a cause of confusion not just for me but for many, judging by the twelve or so issues previously lodged (or an error message stating the dependency is required when running the app - the error message is currently something about keys unable to be exchanged etc but running nm-applet fixes the problem).

Unfortunately, this dependency causes my system to fail to suspend (it won't lock, which is a security concern) and upon opening the lid some of my libinput devices appear to not respond. Looking though `journalctl` it appeared that the vpn was not freeing libraries correctly and it was traced back to nm-applet. Others have lodged similar issues over the lifetime of the library apparently.

It seems like nm-applet should not be required for protonvpn-cli to function correctly - would it not make sense to either treat the dependency as a legitimate, recognised dependency or, preferably, remove it altogether (or use it only if present)?

**Comment by Anglodel** [comment]
It is useful on some WM like i3, to connect to a new secured connection or network. 
It is sometimes buggy, and maybe another program could be used for this, but it’s not an unused dependency.

**Comment by monomadic** [comment]
> it’s not an unused dependency.

I did say undeclared. Downloading the protonvpn package does not ensure nm-applet is functional or even present. nm-applet has its own deps too which I don't want to bring into my system.

**Comment by ferroson** [comment]
> 

Yes, the problem is that nm-applet is not marked as a dependency during installation of protonvpn-cli. As users we don't know if it's needed (when the command fails, we don't know what the problem is and have to find a solution on our own). Even if nm-applet is installed, we don't know that it has to run when using protonvpn-cli.

**Comment by pweibert** [comment]
I love protonvpn, but have the same problem here on Lubuntu. 
How about an alternate authentication mechanism other then using the keyring?
It would not just solve the nm-applet problem but also allow one to connect linux machines to protonvpn before user login. That would really solve a lot of problems for me

**Comment by monomadic** [comment]
This should be filed as a bug at this point.
